<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>932</width>
    <height>672</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CloudBackup</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#centralwidget {
    border-image: url(:/images/background.png) 0 0 0 0 stretch stretch;
}

/* 按钮单独写样式，不设置background: transparent */
QPushButton {
    background-color: rgba(255,255,255,100); /* 半透明 */
    border: 1px solid gray;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout" stretch="0,3,2,2,20,0">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout" stretch="7,8">
            <item>
             <widget class="QLabel" name="imagelabel">
              <property name="layoutDirection">
               <enum>Qt::LeftToRight</enum>
              </property>
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="titlelabel">
              <property name="styleSheet">
               <string notr="true">font-family: '微软雅黑', 'Arial', sans-serif;
font-size: 26px;
font-weight: 700;
color: #3ECFFF;
border-radius: 7px;
padding: 15px 10px;
text-align: center;
letter-spacing: 2px;

box-shadow: 0 2px 8px rgba(0,0,0,0.15);
</string>
              </property>
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0">
            <item>
             <widget class="QLabel" name="backuptimelabel1">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="styleSheet">
               <string notr="true">font-family: &quot;Microsoft YaHei&quot;;
font-size: 14px;
color: #333333;</string>
              </property>
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="backuptimelabel2">
              <property name="styleSheet">
               <string notr="true">font-family: &quot;Microsoft YaHei&quot;;
font-size: 14px;
font-weight: bold;
color: #007ACC;
qproperty-alignment: 'AlignLeft | AlignVCenter';</string>
              </property>
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,1">
            <item>
             <widget class="QPushButton" name="uploadbutton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="styleSheet">
               <string notr="true">/* 基本按钮样式 */
QPushButton {
    /* 渐变背景：从亮蓝到深蓝 */
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                stop:0 #5dade2, stop:1 #2874a6);
    border: none;
    border-radius: 8px;           /* 圆角 */
    color: #ffffff;               /* 字体颜色 */
    font-size: 16px;              /* 字体大小 */
    font-weight: bold;            /* 字体加粗 */
    padding: 8px 20px;            /* 内边距，上下8px，左右20px */
    min-width: 100px;             /* 最小宽度 */
    outline: none;                /* 取消焦点虚线框 */
}

/* 悬停时的效果 */
QPushButton:hover {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                stop:0 #85c1e9, stop:1 #2e86c1);
}

/* 按下时的效果 */
QPushButton:pressed {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                stop:0 #2471a3, stop:1 #1b4f72);
    padding-left: 22px;  /* 模拟按下偏移 */
    padding-top: 10px;
}

/* 禁用时的效果 */
QPushButton:disabled {
    background: #bbb;
    color: #666;
}

/* 如果想在按钮里显示图标+文字，同时图标在左，文字在右： */
QPushButton {
    qproperty-icon: url(:/images/upload.svg);
    qproperty-iconSize: 24px 24px;
    /* 图标和文字自动有间隔 */
}
</string>
              </property>
              <property name="text">
               <string>上传备份文件</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="flushfilelistbutton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="styleSheet">
               <string notr="true">/* 基本按钮样式 */
QPushButton {
    /* 渐变背景：从亮蓝到深蓝 */
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                stop:0 #5dade2, stop:1 #2874a6);
    border: none;
    border-radius: 8px;           /* 圆角 */
    color: #ffffff;               /* 字体颜色 */
    font-size: 16px;              /* 字体大小 */
    font-weight: bold;            /* 字体加粗 */
    padding: 8px 20px;            /* 内边距，上下8px，左右20px */
    min-width: 100px;             /* 最小宽度 */
    outline: none;                /* 取消焦点虚线框 */
}

/* 悬停时的效果 */
QPushButton:hover {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                stop:0 #85c1e9, stop:1 #2e86c1);
}

/* 按下时的效果 */
QPushButton:pressed {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                stop:0 #2471a3, stop:1 #1b4f72);
    padding-left: 22px;  /* 模拟按下偏移 */
    padding-top: 10px;
}

/* 禁用时的效果 */
QPushButton:disabled {
    background: #bbb;
    color: #666;
}

/* 如果想在按钮里显示图标+文字，同时图标在左，文字在右： */
QPushButton {
    qproperty-icon: url(:/images/upload.svg);
    qproperty-iconSize: 24px 24px;
    /* 图标和文字自动有间隔 */
}
</string>
              </property>
              <property name="text">
               <string>刷新文件列表</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QProgressBar" name="progressBar">
            <property name="value">
             <number>24</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTreeWidget" name="backupfilelist">
            <property name="styleSheet">
             <string notr="true">/* 整体表格背景、边框、交替行色 */
QTreeWidget {
    background-color: #FFFFFF;
    border: 1px solid #DDD;
    alternate-background-color: #F9F9F9;
    show-decoration-selected: 1; /* 选中时也显示背景色 */
}

/* 每一行的项 */
QTreeWidget::item {
    padding: 4px;
}

/* 选中行的样式 */
QTreeWidget::item:selected {
    background-color: #CCE4FF;
    color: #000000;
}

/* 表头样式 */
QHeaderView::section {
    background-color: #F0F0F0;
    padding: 6px;
    border: 1px solid #DDD;
    font-family: &quot;Microsoft YaHei&quot;;
    font-size: 14px;
    color: #333333;
}
</string>
            </property>
            <column>
             <property name="text">
              <string>文件名</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>备份时间</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>文件大小</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>下载</string>
             </property>
            </column>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="resultlabel">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="loginpage">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,2,1">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2" stretch="100,30,20,20,20,100">
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="title">
              <property name="styleSheet">
               <string notr="true">font-family: '微软雅黑', 'Arial', sans-serif;
font-size: 26px;
font-weight: 700;
color: #3ECFFF;
border-radius: 7px;
padding: 15px 10px;
text-align: center;
letter-spacing: 2px;

box-shadow: 0 2px 8px rgba(0,0,0,0.15);
qproperty-alignment: 'AlignCenter';</string>
              </property>
              <property name="text">
               <string>欢迎使用云备份系统</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="2,9">
              <item>
               <widget class="QLabel" name="username">
                <property name="styleSheet">
                 <string notr="true">font-family: '黑体';
font-weight: bold;
qproperty-alignment: 'AlignCenter';</string>
                </property>
                <property name="text">
                 <string>账号</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="usernamelineedit">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="2,9">
              <item>
               <widget class="QLabel" name="password">
                <property name="styleSheet">
                 <string notr="true">font-family: '黑体';
font-weight: bold;
qproperty-alignment: 'AlignCenter';</string>
                </property>
                <property name="text">
                 <string>密码</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="passwordlineedit">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="echoMode">
                 <enum>QLineEdit::Password</enum>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QPushButton" name="loginbutton">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="styleSheet">
                 <string notr="true">/* 基本按钮样式 */
QPushButton {
    /* 渐变背景：从亮蓝到深蓝 */
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                stop:0 #5dade2, stop:1 #2874a6);
    border: none;
    border-radius: 8px;           /* 圆角 */
    color: #ffffff;               /* 字体颜色 */
    font-size: 16px;              /* 字体大小 */
    font-weight: bold;            /* 字体加粗 */
    padding: 8px 20px;            /* 内边距，上下8px，左右20px */
    min-width: 100px;             /* 最小宽度 */
    outline: none;                /* 取消焦点虚线框 */
}

/* 悬停时的效果 */
QPushButton:hover {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                stop:0 #85c1e9, stop:1 #2e86c1);
}

/* 按下时的效果 */
QPushButton:pressed {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                stop:0 #2471a3, stop:1 #1b4f72);
    padding-left: 22px;  /* 模拟按下偏移 */
    padding-top: 10px;
}

/* 禁用时的效果 */
QPushButton:disabled {
    background: #bbb;
    color: #666;
}

/* 如果想在按钮里显示图标+文字，同时图标在左，文字在右： */
QPushButton {
    qproperty-icon: url(:/images/upload.svg);
    qproperty-iconSize: 24px 24px;
    /* 图标和文字自动有间隔 */
}
</string>
                </property>
                <property name="text">
                 <string>登录</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="registerbutton">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="styleSheet">
                 <string notr="true">/* 基本按钮样式 */
QPushButton {
    /* 渐变背景：从亮蓝到深蓝 */
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                stop:0 #5dade2, stop:1 #2874a6);
    border: none;
    border-radius: 8px;           /* 圆角 */
    color: #ffffff;               /* 字体颜色 */
    font-size: 16px;              /* 字体大小 */
    font-weight: bold;            /* 字体加粗 */
    padding: 8px 20px;            /* 内边距，上下8px，左右20px */
    min-width: 100px;             /* 最小宽度 */
    outline: none;                /* 取消焦点虚线框 */
}

/* 悬停时的效果 */
QPushButton:hover {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                stop:0 #85c1e9, stop:1 #2e86c1);
}

/* 按下时的效果 */
QPushButton:pressed {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                stop:0 #2471a3, stop:1 #1b4f72);
    padding-left: 22px;  /* 模拟按下偏移 */
    padding-top: 10px;
}

/* 禁用时的效果 */
QPushButton:disabled {
    background: #bbb;
    color: #666;
}

/* 如果想在按钮里显示图标+文字，同时图标在左，文字在右： */
QPushButton {
    qproperty-icon: url(:/images/upload.svg);
    qproperty-iconSize: 24px 24px;
    /* 图标和文字自动有间隔 */
}
</string>
                </property>
                <property name="text">
                 <string>注册</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>932</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
